### PR TITLE
Updated layout of dialogs and updated labels

### LINKED ISSUE #420: Improve presentation of cancel dialog
It currently looks like this:

image

There's a bunch of issues here:

 * The title is "MediaWriter", which isn't helpful
 * There's a close button in the title bar - this is non-standard for a confirmation dialog, and its purpose will be unclear (I don't see a way to avoid this with Qt Quick dialogs though)
 * Typically the format of a confirmation dialog is to have a heading with a longer description below; the lack of heading makes the window harder to read
 * The layout isn't very pleasing - it mixes centered alignment for the text, with right alignment for the buttons, and the margins are off
 * The dialog doesn't appear to be modal - its position over its parent isn't fixed, and the parent doesn't appear non-interactive

This is a tricky one, given that Windows, Mac and GNOME have different conventions for confirmation dialogs, plus there are constraints to what we can do with Qt Quick. Given that both Windows and Mac use left/right alignment for their confirmation dialogs, perhaps a layout like this might work best overall?

image

### LINKED ISSUE #421: Various minor string issues
image

"Select drive" should use header capitalization - "Select Drive"

image

"Write" should maybe be "Download & Write"? It stuck me as being a bit misleading to click "write", when that part of the operation won't happen immediately.

image

"Back" and "Next" aren't logical opposites. It should either be "Previous" and "Next", or "Back" and "Forward" (the mockups used "Previous" and "Next").

image

Spurious space between "Select" and "...".

image

"The file will be saved to your Downloads folder." Which file? Also, it's better to avoid addressing the user directly. "Downloads are saved to the downloads folder."

### LINKED ISSUE #423: Heading says "downloading" while disk is being written
image

The heading should update to say `Writing <Fedora Version>` or `Writing <Fedora Version> to Disk`.

### PR DESCRIPTION
**Pull request by gastoner**
Fixes #420 
Fixes #421 
Fixes #423

### CODE REVIEW

**Review by grulja — CHANGES_REQUESTED** [review]

**Comment by gastoner** [comment]
Fixes #278

File: src/app/qml/CancelDialog.qml [hunk 1]
```diff
@@ -32,34 +32,44 @@ ApplicationWindow {
...
+                        qsTr("Cancel Verification?")
+                }  
+            }
+            
             Label {
-                Layout.alignment: Qt.AlignVCenter
                 text: {
                     if (releases.variant.status == Units.DownloadStatus.Downloading || releases.variant.status === Units.DownloadStatus.Download_Verifying)
-                        qsTr("Downloading is still in progress, do you wish to cancel it?")
+                        qsTr("Download progress will be lost and media writing will be aborted?")
```
Comment (grulja) [inline src/app/qml/CancelDialog.qml:65]: This shouldn't end with a questionmark.

File: src/app/qml/CancelDialog.qml [hunk 2]
```diff
@@ -32,34 +32,44 @@ ApplicationWindow {
...
                     else if (releases.variant.status == Units.DownloadStatus.Writing) 
-                        qsTr("Do you want to cancel writing?")
+                        qsTr("Writing process will be aborted and your drive will have to be restored afterwards?")
```
Comment (grulja) [inline src/app/qml/CancelDialog.qml:67]: Same here.

File: src/app/qml/DownloadPage.qml [hunk 3]
```diff
@@ -41,22 +41,70 @@ Page {
...
+                    qsTr("%1 Sucessfuly Written").arg((String)(releases.localFile.iso).split("/").slice(-1)[0])
+                else if (mainWindow.selectedOption == Units.MainSelect.Write)
+                    qsTr("Writing %1").arg((String)(releases.localFile.iso).split("/").slice(-1)[0])
+                else if (releases.variant.status === Units.DownloadStatus.Write_Verifying || releases.variant.status === Units.DownloadStatus.Writing)
+                    qsTr("Writing %1 ").arg(releases.selected.name) + releases.selected.version.number
+                else if (releases.variant.status === Units.DownloadStatus.Download_Verifying || releases.variant.status === Units.DownloadStatus.Downloading)
+                    qsTr("Downloading %1 ").arg(releases.selected.name) + releases.selected.version.number
+                else if (releases.variant.status === Units.DownloadStatus.Finished)
+                    qsTr("%1 Sucessfuly Written").arg(releases.selected.name + " " + releases.selected.version.number)
+
```
Comment (grulja) [inline src/app/qml/DownloadPage.qml:55]: Remove redundant new line

**Review by grulja — CHANGES_REQUESTED** [review]

**Pull request merged**